### DISCUSSION TITLE
Quarto

### DISCUSSION THREAD

**Discussion by trey-stafford** [nsidc/tech-radar#9] [Tools]
[Quarto](https://quarto.org/) is an open-source scientific and technical publishing system.

Currently utilized for producing content (website and RevealJS slide decks) for QGreenland's [Researcher](https://github.com/qgreenland-workshop-2023-researcher/qgreenland-workshop-2023-researcher.github.io) and [Educator](https://github.com/qgreenland-workshop-2023-educator/qgreenland-workshop-2023-educator.github.io) workshops.

**Comment by MattF-NSIDC** [comment]
FWIW I'm maintaining the Quarto conda-forge [feedstock](https://github.com/conda-forge/quarto-feedstock) on my personal time, so you can pester me if you need a new release!
